docs/use.md

```diff
@@ -187,6 +190,119 @@ The [render-as-you-fetch pattern](https://reactjs.org/docs/concurrent-mode-suspe
 
 Just as the SDK hooks like `useFirestore` can automatically fetch an SDK, you can call `preloadFirestore` (or `preloadAuth`, etc) to start loading an SDK without suspending.
 
+### Initialize an SDK
+
+A few Firestore SDKs need to be initialized (`firebase().performance().initialize()`, `firebase.remoteConfig().fetchAndActivate()`), or need to have settings set before any other calls are made (`firebase.firestore().enablePersistence()`). This can be done by using an extra argument in the preload method:
```

Comment:
Drop perf

Reply:
We should mention that if the function sent to the second arg returns a Promise it awaits otherwise it continues immediately, (e.g, make sure to return `enablePersistence()`/`fetchAndActivate()` etc. rather than a void to ensure the effect happens)

Reply:
Now that I'm thinking about it an API change might make more sense. We should optimize for a second app being rare.

`preloadFirestore({ firebaseApp, setup })` or `preloadFirestore(app => app.firestore().enabledPersistence())` the later one being us passing the contextual app; but they could ignore it `preloadFirestore(() => myApp.firestore())`.

Another interesting thought here, especially w/re RC; how are we ensuring that the setup setup is complete before `useRemoteConfig` emits? e.g, `fetchAndActivate`

Reply:
I dropped perf and added `firebaseApp` to the `preloadFirestore` call. Also opened #217 to discuss API change